src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -124,6 +124,15 @@ public SnowflakeSinkTask(
     this.conn = connectionService;
   }
 
+  @VisibleForTesting
+  public SnowflakeSinkTask(
+      SnowflakeSinkService service,
+      SnowflakeConnectionService connectionService,
+      Map<String, String> topic2table) {
+    this(service, connectionService);
+    this.topic2table = topic2table;
+  }
+
```

Comment:
FYI Codecov is claiming lines 134-135 are not covered via tests.

Reply:
Thanks, trying out a couple things to ignore those lines

Reply:
Now it says lines 135-136 :)

Reply:
code coverage check isn't complaining anymore, resolving